### ISSUE TITLE
Align language / time section so they are all aligned to the bottom like the ones on the right

### ISSUE THREAD

**Issue by seanauriti** [CoderBounty/coderbounty#117]
![image](https://cloud.githubusercontent.com/assets/7475382/11455305/bb8f257e-9625-11e5-8e72-61d60aae7e26.png)

**Comment by seanauriti** [comment]
related to #143

**Comment by coderbounty-cowboy** [comment]
Coderbounty posted: http://coderbounty.com/issue/62

**Comment by seanauriti** [comment]
Address this in new design.  Remove bounty...

**Comment by lkd70** [comment]
Just took a look at this, new to the site so thought I'd see what's what... It occurs to me no matter how you do it with this current fixed format you'll always get overlapping or missing content (overflowing a div...)

I tried to play with the CSS a little, solutions I came up with (as simple as possible)
1. Move code language logo to side-info div:
https://vgy.me/dnLxil.png
2. Edit posted class CSS:
```
position: absolute;
bottom: 0;
```
(also, replaced the space in between posted and views UL's to a break, as I feel these elements don't look right together, as if the posted or views is too long it causes wrapping.)
https://vgy.me/pofHWf.png

Not really a solution, here's an example of the wrapping issue caused:
https://vgy.me/aSSBpd.png
I targeted it by adding a class to the p elements with this css:
```
    overflow: hidden;
    text-overflow: ellipsis;
    display: -webkit-box;
    -webkit-box-orient: vertical;
    -webkit-line-clamp: 5;
    line-height: 100%;
    word-break: break-word;
```
This also fixes the use of ReallyLongWordsOrURLsThatWillNotWrapAround due to the use of word-break.
Example: https://vgy.me/F3GnlC.png

**Comment by seanauriti** [comment]
Looks good, thank you!  Feel free to submit a pull request and I'll merge it.

**Comment by seanauriti** [comment]
still getting some overflow here:
![image](https://cloud.githubusercontent.com/assets/7475382/19128382/1dafe938-8b11-11e6-92d1-493f2c77a7a6.png)

**Comment by seanauriti** [comment]
also would be good if the title "#109 | coderbounty" could be on one line.

**Comment by lkd70** [comment]
I've addressed this issue in another issue post at #213
